### ISSUE TITLE
Pipeline is failing when it throws a warning

### ISSUE THREAD

**Issue by GertjanRoke** [slavcodev/coverage-monitor-action#51]
I have a `threshold_alert` set to `0` and a `threshold_warning` set to `75` as you can see below, but for some reason the pipeline now fails instead of passing.

Now:
![image](https://user-images.githubusercontent.com/6010509/155134151-94d0896c-2717-443d-8bb7-642cc1eb10fc.png)

Yesterdag:
![image](https://user-images.githubusercontent.com/6010509/155134221-e3a1f36d-3003-490c-9fde-be235659af85.png)

```yaml
- name: Report coverage to Pull Request
   uses: slavcodev/coverage-monitor-action@v1
   if: ${{ github.event_name == 'pull_request' }}
   continue-on-error: true
   with:
      github_token: ${{ secrets.GITHUB_TOKEN }}
      coverage_path: coverage-clover.xml
      threshold_alert: 0
      threshold_warning: 75
      comment: false
```

Looks to be because of the last release, if you need more info please let me know, would love to help.

**Comment by GertjanRoke** [comment]
Just set `threshold_alert` to `1` and then it works fine, if this is expected then you can close this issue 😉

**Comment by slavcodev** [comment]
Nope, it's not expected, it's a bug. 

Just fixed it in v1.4.1, should already work for you if you use generic version `v1`.

Thank you, @GertjanRoke
